Developers/Newcomers_Guide.rst

```diff
@@ -3,15 +3,20 @@
 Welcome to the Newcomers Guide!
 ===============================
 
-This is a step-based guide that will help you get your first contribution
-at coala!
+To become part of the coala developers team, there's a few steps you need
+to complete. The newcomer process is as it follows:
 
-There's a few things a contributor needs in order to be considered a full-norm
-coalaian:
+You will start as a newcomer, which is kind of a trial. If you complete the
+following tasks, you will become a developer at coala:
 
   - merge a ``difficulty/newcomer`` Pull Request
-  - review other Pull Requests
+  - review at least a ``difficulty/newcomer`` Pull Request
   - merge a ``difficulty/low`` Pull Request
+  - review at least a``difficulty/low`` or higher Pull Request
+
+Once you have achieved all these and the maintainers consider that you have
+helped with reviewing as it is stated, you will be promoted to the developers
+team.
```

Comment:
maybe something like "Just ask for being promoted on the chat and provide links to your reviews and merged Pull Requests!"

Also we should add a benefit for being a developer. Maybe rephrase the last part of this sentence to "you will be able to name yourself a coala developer", it's kind of "now you're accepted" but polite